internal/liquids/ironic/liquid.go

```diff
@@ -37,8 +37,9 @@ import (
 
 type Logic struct {
 	// configuration
-	WithSubcapacities bool `json:"with_subcapacities"`
-	WithSubresources  bool `json:"with_subresources"`
+	WithSubcapacities bool                               `json:"with_subcapacities"`
+	WithSubresources  bool                               `json:"with_subresources"`
+	NodeToAZOvewrites map[string]liquid.AvailabilityZone `json:"node_to_az_overwrites"`
```

Comment:
Typo:
```suggestion
	NodeToAZOverwrites map[string]liquid.AvailabilityZone `json:"node_to_az_overwrites"`
```

Also, you probably mean "override" instead of "overwrite".

Also also, please add one line of comment that this is to cover legacy deployments.

Reply:
Yes, I included the mentioned points.